app/assets/stylesheets/nav-bar.scss

```diff
@@ -23,23 +23,20 @@
         text-decoration:none;
         font-weight:$regular;
         position:relative;
-        &:before {
```

Comment:
Why has this block been removed?

Reply:
It didn't serve any purpose as far as I could tell and I didn't want it to cause any conflict with the new approach.

Reply:
It underlines the link. 

Before: 
<img width="241" alt="screenshot 2019-01-23 at 11 42 10" src="https://user-images.githubusercontent.com/286476/51632356-f5c1ea00-1f03-11e9-91c6-6367e685f718.png">

After:
<img width="261" alt="screenshot 2019-01-23 at 11 42 13" src="https://user-images.githubusercontent.com/286476/51632371-fce8f800-1f03-11e9-80d7-115fee57b213.png">

Reply:
Good catch. I restored that code.